docs/running-locally.md

```diff
@@ -1,22 +1,25 @@
 # Running Locally
-## Pre-requisites:
+
+## Pre-requisites
 
 * Golang
 * Yarn. `brew install yarn` 
 * Docker
 * [Kustomize](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/INSTALL.md)
 * [protoc](http://google.github.io/proto-lens/installing-protoc.html) `brew install protobuf`
 * `jq`
-* Kubernetes Cluster (we recommend Docker for Desktop + K3D, as this will allow you to test RBAC set-up, and is also fast)
-
-Useful:
-
-* For a PS1 prompt showing your current kube context: kube-ps1 to help.  `brew install kube-ps1`
+* A local Kubernetes cluster
 
-K3D tip: You can set-up K3D to be part of your default kube config as follows
+We recommend using [K3D](https://k3d.io/) to set up the local Kubernetes cluster since this will allow you to test RBAC set-up and is fast. You can set-up K3D to be part of your default kube config as follows:
 
     cp ~/.kube/config ~/.kube/config.bak
     cat $(k3d get-kubeconfig --name='k3s-default') >> ~/.kube/config
+    
+Alternatively, you can use [Minikube](https://github.com/kubernetes/minikube) to set up the local Kubernetes cluster. Once a local Kubernetes cluster has started via `minikube start`, your kube config will use Minikube's context automatically.
+
+Useful:
+
+* For a PS1 prompt showing your current kube context: kube-ps1 to help.  `brew install kube-ps1`
```

Comment:
Could you explain why this is needed?

Reply:
This was in the doc before. I am not sure why it's necessary either. Should I just delete it?

Reply:
simpler is better - > please delete

Reply:
Thanks. Done.